docs/extend/extend-apiml/api-mediation-oidc-authentication.md

```diff
@@ -83,6 +83,28 @@ API ML provides a Zowe CLI plugin to help administrators generate a JCL for crea
 
   See the [Identity Federation cli plugin](../../user-guide/cli-idfplugin.md) documentation for details about how to use the plugin tool to set up the mapping in the ESM of your z/OS system.
 
+Once the commands are generated from the IDF(Identity Federation cli plugin) tool, admin can run them on corresponding ESMs to create disrtibuted filters.
+
+- RACF :
+RACMAP ID(userid) MAP USERDIDFILTER(NAME('distributed-identity-user-name')) REGISTRY(NAME('distributed-identity-registry-name' )) WITHLABEL('label-name')
+
+SETROPTS RACLIST(IDIDMAP) REFRESH
+
+- ACF2 :
+ACF
+SET PROFILE(USER) DIVISION(IDMAP)
+INSERT userid.ZWEDNMAP IDMAPDN(distributed-identity-user-name) -
+IDMAPRN(distributed-identity-registry-name) IDLABEL(label-name)
+
+F ACF2,REBUILD(USR),CLASS(P),DIVISION(IDMAP)
+END
+
+- TSS :
+  TSS ADD(userid) IDMAP(ZWEDNMAP) IDMAPDN('distributed-identity-user-name') -
+  IDMAPRN('distributed-identity-registry-name') IDLABEL('label-name')
+
+  TSS REFRESH
```

Comment:
See below there are items for RACF, TSS and ACF2 (also remove the CA prefix). These commands should be included in their respective item.